corehq/apps/userreports/templates/userreports/partials/report_builder_configure_report.html

```diff
@@ -49,6 +50,31 @@
                 )
             }
 
+            this.dataType = ko.computed(function() {
+                // Returns type of Question, applicable only if optionsContainQuestions is true
+                var property = this.property()
+                var propertyObject = _.find(this.propertyOptions(), function(option){
+                    return option.value == property;
+                });
+                if (propertyObject !== undefined && propertyObject.hasOwnProperty("type"))
+                    return propertyObject.type;
+                else
+                    return undefined;
+            }, this);
+
+            this.calculationOptions = ko.observableArray(defaultCalculationOptions);
+            if (this.optionsContainQuestions) {
+                // If options are questions, calculationOptions should be limited based on the dataType
+                this.calculationOptions = ko.computed(function() {
+                    if (this.dataType() == "Int") {
```

Comment:
Hey, for some reason I thought all the numeric types will have 'Int' as the type. Will update it.

Will also update to make hidden values have all format options.